### PR TITLE
Improve configs for viewing test coverage

### PR DESCRIPTION
**Pull request by yu-smc**
-  add jest config to see test coverages layer by layer
-  add 'coverage' command which evlaluates jest coverage, and then open the summary page on the browser

*Running 'yarn add' is required

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,11 +24,13 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "test": "jest",
+    "coverage": "jest --coverage && browser-sync start --server \"coverage/lcov-report\"",
```
Comment (t29mato) [inline package.json:27]: The script combines the jest coverage command with the browser-sync server command. Splitting these into separate scripts could offer more flexibility. For instance, developers might want to generate coverage without serving it on a browser or serve an existing report without regenerating it.
So, what about refactoring the scripts like this:

```
"scripts": {
  "test:coverage": "jest --coverage",
  "coverage:report": "browser-sync start --server \"coverage/lcov-report\"",
  "coverage": "npm run test:coverage && npm run coverage:report"
}

```

File: jest.config.cjs
Comment (t29mato) [inline jest.config.cjs:1]: I noticed the comment "exclude other unnecessary files" in the config. Could you also add a note explaining why those files are considered unnecessary? I mean, I checked the folder and saw it's just an index.ts file in there, so I get why you might exclude it. But it'd be super helpful if the reason was clear right in the comments, you know, so anyone can understand without having to dig through the folders.

**Review by t29mato — COMMENTED** [review]
Thank you for the PR. I was able to review it more thoroughly since the changes were minimal. I have a few non-essential comments, but I would appreciate your addressing them.

File: package.json (hunk 1 above)
Comment (yu-smc) [reply package.json:27]: Thank you, I completely agree with your idea of refactoring so will change package json in the next PR!

File: jest.config.cjs
Comment (yu-smc) [reply jest.config.cjs:1]: My comment was indeed a bit vague.
Commenting the detailed reason of excluding files from coverage is a little too much to me, so I am thinking of fixing comment to such as 'Exclude files that is not necessary to test' or so to keep 
versatility🏃‍♀️

**Comment by yu-smc** [comment]
@t29mato 

Thank you for your review.
I opened a PR of refactoring based on your comments.
Approving is not necessary for merge, but please look through if you have time

https://github.com/t29mato/starry-digitizer/pull/127/files

**Pull request merged**